acceptance/tests/resource/service/ticket_3927_redhat_systemd_vs_service_provider.rb

```diff
@@ -0,0 +1,46 @@
+test_name 'RedHat Service vs. Systemd Provider Validation'
+
+# A simple acceptance test to ensure basic usage of the service
+# provider works for a mix of sysV and systemd RedHat Linux platforms
+confine :to, :platform => /el|centos|fedora/
+
+manifest_install_httpd = %Q{
+  package { 'httpd':
+    ensure => present,
+  }
+}
+
+manifest_httpd_enabled = %Q{
+  service { 'httpd':
+    enable => true,
+  }
+}
+
+manifest_httpd_disabled = %Q{
+  service { 'httpd':
+    enable => false,
+  }
+}
+
+agents.each do |agent|
+  distro = on(agent, facter('operatingsystem')).stdout.chomp
+  if distro == 'fedora'
+    majrelease = on(agent, facter('operatingsystemmajrelease')).stdout.chomp.to_i
+    if majrelease < 17
+      skip_test "Test not applicable to Fedora #{majrelease}"
+    elsif majrelease > 20
```

Comment:
I don't see why checking for `majrelease > 21` here would cause CI failures, unless I'm misunderstanding the purpose of this code.  When we do stand up a F21 config, won't it unnecessarily fail if we leave it `> 20`?